Planner/src/Planner.java

```diff
@@ -1,26 +1,35 @@
 import java.awt.BorderLayout;
 import java.awt.Color;
+import java.awt.Dimension;
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
 
+import javax.swing.BorderFactory;
 import javax.swing.Box;
 import javax.swing.BoxLayout;
 import javax.swing.DefaultListModel;
 import javax.swing.JButton;
+import javax.swing.JComboBox;
 import javax.swing.JFrame;
 import javax.swing.JLabel;
 import javax.swing.JList;
 import javax.swing.JOptionPane;
 import javax.swing.JPanel;
 import javax.swing.JTextField;
+import javax.swing.SwingConstants;
+import javax.swing.border.Border;
 
 public class Planner extends JFrame {
 	JFrame plannerFrame = new JFrame();
 	JPanel listPanel = new JPanel();
 	JPanel topPanel = new JPanel();
-	JPanel panelButtons = new JPanel();
+	JPanel buttonPanel = new JPanel();
 	JButton addUserButton = new JButton("Add new User");
 	JButton addTaskButton = new JButton("Add new Task");
+	JButton selectTaskButton = new JButton("Choose new task");
```

Comment:
It's a very good idea this one ! It allows us to assign a task to a User. I would change the name of the button maybe in Assign Task or something similar.